slash/css/src/Tag/Tag.scss

```diff
@@ -3,18 +3,27 @@
 @use "../common/common" as common;
 
 $types: (
-  success common.$color-malachite,
-  info common.$color-blue-alert,
-  danger common.$color-orange,
-  error common.$color-pink
+  success #0c7d3b,
```

Comment:
it is better to use the color token everywhere, if they are not present, add them

Reply:
I see this with @MartinWeb, They will be added later on Token file